packages/web-components/@sfx/product/stories/index.ts

```diff
@@ -0,0 +1,103 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import '../src/index.ts';
```

Comment:
The extension is not needed.

```suggestion
import '../src/index';
```